### PR TITLE
Added editing policy

### PR DESCRIPTION
**Pull request by hauschke**
**[VIVO-Ontology GitHub issue](https://github.com/vivo-ontologies/vivo-ontology/issues)**:  
* https://github.com/vivo-ontologies/vivo-ontology/issues/50

* Other relevant links (mailing list discussion, related pull requests, etc.)
* https://wiki.lyrasis.org/display/VIVO/2024-08-21+Ontology+Interest+Group+Call

# What does this pull request do?
Adds an editing policy to the readme to only allow ontology edits with Protégé.

# Interested parties
@vivo-ontologies/team-members

### CODE REVIEW

**Comment by hauschke** [comment]
It was clarified in today's onto call that there is a need for some testing. @brianjlowe volunteered to set up a repo for that purpose.

Decision documented in https://wiki.lyrasis.org/display/VIVO/2024-09-04+Ontology+Interest+Group+Call.

**Comment by hauschke** [comment]
Decision in https://wiki.lyrasis.org/display/VIVO/2024-09-04+Ontology+Interest+Group+Call: Tests have to be performed. https://github.com/hauschke/vivo-ontology/tree/owlapi_test was created. 

First test successful: https://github.com/hauschke/vivo-ontology/pull/1

**Comment by hauschke** [comment]
2nd test by @StroemPhi: https://github.com/hauschke/vivo-ontology/pull/2

It is important to do a proper reserialization. The PR for this needs to be carefully checked.

**Comment by hauschke** [comment]
Decision in https://wiki.lyrasis.org/display/VIVO/2024-10-02+Ontology+Interest+Group+Call:

- Create recommendations for users:
-- Use Protege for big modifications and re-serialize the result before creating a PR with robot tool to minimize difference to review.
-- For small modifications when modification can be done in text editor no re-serialization is needed.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,15 @@ VIVO Ontologies is an open source community. All are welcome to open issues, par
 
 You can contribute using the fork, branch and pull model. Helpful descriptions of the process are available [on the VIVO wiki](https://wiki.lyrasis.org/display/VIVO/Contributing+code+with+a+fork%2C+branches%2C+and+pull+requests).
 
+## Editing Policy
+Manual structural changes can result in inconsistencies. Editing and saving the file with ontology editors or IDES can lead to new serialisations, which creates verbose diffs. This makes reviewing the changes very difficult. Therefore we recommend the following approaches for editing:
```
Comment (chenejac) [inline README.md:22]:
```suggestion
Manual structural changes can result in inconsistencies. Editing and saving the file with ontology editors or IDEs can lead to new serialisations, which creates verbose diffs. This makes reviewing the changes very difficult. Therefore we recommend the following approaches for editing:
```

**Review by chenejac — APPROVED** [review]
@hauschke looks very good. Please, consider my tiny suggestion before this PR is merged.

**Review by brianjlowe — APPROVED** [review]

**Pull request merged**